src/jaegertracing/DynamicLoad.cpp

```diff
@@ -19,21 +19,23 @@
 
 #include <opentracing/dynamic_load.h>
 
-#include "TracerFactory.h"
 #include "jaegertracing/Tracer.h"
+#include "jaegertracing/TracerFactory.h"
 
-int OpenTracingMakeTracerFactory(const char* opentracingVersion,
-                                 const void** errorCategory,
-                                 void** tracerFactory)
+static int makeTracerFactory(const char* opentracingVersion,
+                             const char* opentracingABIVersion,
+                             const void** errorCategory,
+                             void* errorMessage,
+                             void** tracerFactory)
 {
     assert(errorCategory != nullptr);
     assert(tracerFactory != nullptr);
 #ifndef JAEGERTRACING_WITH_YAML_CPP
     *errorCategory =
-        static_cast<const void*>(&opentracing::dynamic_load_error_category());
+        static_cast<const void*>(&opentracing::dynamic_load_errorCategory());
```

Comment:
dynamic_load_errorCategory?

Reply:
Lol I was camelcasing this stuff an must have messed that up. Surprised this built...